### PR TITLE
Add worker JS cache-busting + Change all tape lines to green

### PR DESCRIPTION
**Pull request by charleshong3**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -13,8 +13,11 @@
 
     <title>PiE Robotics Simulator</title>
 
+    <!-- Store git_hash -->
+    <script>var gitHash = "{{ git_hash }}";</script>
+
     <!-- Disables scrolling with arrow keys and spacebar -->
-    <script src="/static/js/disable_scrolling.js?t={[ git_hash }}"></script>
+    <script src="/static/js/disable_scrolling.js?t={{ git_hash }}"></script>
```
Comment (quantumfusion) [inline templates/index.html:20]: Ah, good catch here.

File: static/js/robot.js [hunk 2]
```diff
@@ -22,12 +22,13 @@ var console=(function(oldCons){
     };
 }(console));
 
+var queryString = location.search;
```
Comment (quantumfusion) [inline static/js/robot.js:26]: This is fine, I generally prefer (style-wise) to be more specific on what gets parsed out of `location.search`, but  this is fine.

**Review by quantumfusion — DISMISSED** [review]

File: static/js/robot.js (hunk 2 above)
Comment (charleshong3) [reply static/js/robot.js:26]: I don't know if I necessarily resolved this, but I added a comment that hopefully helps

**Review by kevinzhu00 — APPROVED** [review]
Looks good.

**Pull request merged**